src/profiler/screeps-profiler.js

```diff
@@ -419,14 +418,14 @@ const Profiler = {
     },
 
     shouldPrint() {
-        const streaming = Profiler.type() === 'stream';
-        const profiling = Profiler.type() === 'profile';
+        const streaming = Profiler.type() === "stream";
+        const profiling = Profiler.type() === "profile";
         const onEndingTick = Memory.profiler.disableTick === Game.time;
         return streaming || (profiling && onEndingTick);
     },
 
     shouldEmail() {
-        return Profiler.type() === 'email' && Memory.profiler.disableTick === Game.time;
+        return Profiler.type() === "email" && Memory.profiler.disableTick === Game.time;
     },
 };
 
```

Comment:
remove all changes to this file - the linter may complain otherwise and i prefer single ticks